test/spec/modules/enrichmentFpdModule_spec.js

```diff
@@ -99,4 +99,18 @@ describe('the first party data enrichment module', function() {
     expect(validated.device).to.deep.equal({ w: 1200, h: 700 });
     expect(validated.site.keywords).to.be.undefined;
   });
+
+  it('should store a reference to gpc witin ortb2.regs.ext if it has been enabled', function() {
+    let validated;
+    width = 800;
+    height = 500;
+
+    validated = processFpd({}, {}).global;
+    expect(validated.regs).to.equal(undefined);
+
+    navigator.globalPrivacyControl = { gpc: 1 };
```

Comment:
as I read the proposal this should be `true` or `false`. However, I have two objections: 

-  aren't we doing this too early? none of my browsers set this, and it's not even mentioned on MDN.  Is the proposal final and on its way to implementation?
- Once the navigator _does_ set this, this test will not work right, because it's going to be a read-only value. I think you need to mock it here.

Reply:
just updated my PR.  somehow missed the `true` or `false` part thanks for pointing that out.  i think the test is ok now.

correct, from what i gathered as well, this functionality does still seem to be in it's early stages, not 100% sure when it will be fully implemented.  it does state here how detection will occur though: https://global-privacy-control.glitch.me/ (was taken from https://globalprivacycontrol.org/, from the faq at the bottom under "I’m a publisher, developer, or other service. How can I support GPC?")

i was aiming to be proactive, however, as you mentioned because this is still in the proposal phase should we hold off entirely for the time being in case specifications change?

Reply:
I think it's OK because as it's written the worst that can happen is that we don't pick up the gpc flag. I'm just surprised that we'd get here before MDN :)

Reply:
haha i agree, MDN typically knows all.  least the change on our end for this is pretty minimal/low-risk like you pointed out anyway